config/application.rb

```diff
@@ -44,6 +44,10 @@ class Application < Rails::Application
 
     config.middleware.use Rack::Deflater
 
+    config.public_file_server.headers = {
+      "Cache-Control" => "public, max-age=2147483648"
```

Comment:
Should we add immutable, too?

```suggestion
      "Cache-Control" => "public, max-age=2147483648, immutable"
```